src/function/getJournal.ts

```diff
@@ -4,6 +4,7 @@ import createResponse from '../util/createResponse';
 import journal from '../service/journal';
 
 export function getJournal(event: any, context: Context, callback: Callback) {
+  context.callbackWaitsForEmptyEventLoop = false;
```

Comment:
cool 👍

Reply:
Why do we do this?

Reply:
It's a way of solving the timeout issues we were having timeouts after calling redis.

https://stackoverflow.com/questions/41621776/why-does-aws-lambda-function-always-time-out